tests/routes/view.test.js

```diff
@@ -505,7 +505,7 @@ describe('Routes', () => {
 
   it('bulk matching route should call the case service to fetch case data', async () => {
```

Comment:
pedantic of me sorry, maybe add `to fetch first 20` to the title. idk what hearingOutcomesEnabled: false means

`bulk matching route should call the case service to fetch first 20 cases data`